src/templates/codebuild-deployment-template.yaml

```diff
@@ -495,6 +495,41 @@ Resources:
                       return
 
                   outputs = {o['ExportName']: o['OutputValue'] for o in response['Stacks'][0].get('Outputs', []) if 'ExportName' in o}
+
+                  # Construct exports dashboard URL from CloudFront domain and assets bucket
+                  dashboard_url = None
+                  cloudfront_domain = None
+                  assets_bucket = None
+
+                  # Look for WorkshopCloudFrontDomain export first
+                  for export_name, export_value in outputs.items():
+                      if export_name == 'WorkshopCloudFrontDomain':
+                          if export_value.startswith('https://'):
+                              cloudfront_domain = export_value.rstrip('/')
+                          else:
+                              cloudfront_domain = f"https://{export_value.rstrip('/')}"
+                          break
+
+                  # Find assets bucket
+                  for export_name, export_value in outputs.items():
+                      if 'AssetsBucket' in export_name or 'assets' in export_name.lower():
+                          # Extract bucket name from ARN if needed
+                          if export_value.startswith('arn:aws:s3:::'):
+                              assets_bucket = export_value.split(':::')[-1]
+                          else:
+                              assets_bucket = export_value
+                          break
+
+                  # Construct dashboard URL
+                  if cloudfront_domain:
+                      dashboard_url = f"{cloudfront_domain}/workshop-exports/index.html"
+                  elif assets_bucket:
+                      dashboard_url = f"https://{assets_bucket}.s3.amazonaws.com/workshop-exports/index.html"
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Potential S3 bucket sniping vulnerability detected. This rule has identified S3 bucket references that could be vulnerable to bucket sniping attacks. Bucket sniping occurs when an attacker registers an S3 bucket name after finding it referenced in code but not yet created. This can lead to data exposure, malicious content hosting, or service disruption.

Recommendations:
1. Create all referenced S3 buckets immediately
2. Use organization-specific prefixes for bucket names
3. Verify bucket ownership before use
4. Consider using AWS Organizations S3 bucket naming rules

Discovered: workshop-exports